errors.go

```diff
@@ -12,16 +12,28 @@ var ErrAdminKey = models.SPVError{Message: "an admin key must be set to be able
 // ErrMissingXpriv is when xpriv is missing
 var ErrMissingXpriv = models.SPVError{Message: "xpriv missing", StatusCode: 401, Code: "error-unauthorized-xpriv-missing"}
 
+// ErrMissingXprivAndXpub is when xpriv and xpub are missing
+var ErrMissingXprivAndXpub = models.SPVError{Message: "xpriv and xpub missing", StatusCode: 404, Code: "error-shared-config-xpriv-and-xpub-missing"}
```

Comment:
I think xpriv `or` xpub should be provided. Not both.

Reply:
Came back to old message version